e2e/testcafe-devextreme/tests/editors/checkBox/common.ts

```diff
@@ -47,3 +59,93 @@ fixture.disablePageReloads`CheckBox`
     await createWidget('dxCheckBox', { value: undefined, text: 'indeterminate', rtlEnabled: true }, '#indeterminateRTL');
   });
 });
+
+[20, undefined, 25].forEach((iconSize) => {
+  test('Checkbox configuration, different Checkbox icon sizes', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await testScreenshot(t, takeScreenshot, `CheckBox configurations iconSize=${iconSize === undefined ? 'default' : iconSize}.png`, { element: '#container', shouldTestInCompact: true });
+
+    await testScreenshot(t, takeScreenshot, `CheckBox configurations iconSize=${iconSize === undefined ? 'default' : iconSize}.png`, { element: '#container', theme: getDarkThemeName() });
+
+    await t
+      .expect(compareResults.isValid())
+      .ok(compareResults.errorMessages());
+  }).before(async () => {
+    await setStyleAttribute(Selector('#container'), 'padding: 5px; width: 300px; height: 400px; column-count: 2;');
+
+    await insertStylesheetRulesToPage(`.${CHECKBOX_CLASS} { display: block; }`);
+
+    for (const text of ['label', 'one two three']) {
+      for (const rtlEnabled of [false, true]) {
+        for (const state of [
+          READONLY_STATE_CLASS,
+          DEFAULT_STATE_CLASS,
+          ACTIVE_STATE_CLASS,
+          HOVER_STATE_CLASS,
+          FOCUSED_STATE_CLASS,
+          DISABLED_STATE_CLASS,
+        ] as any[]
+        ) {
+          for (const value of valueModes) {
+            const id = `dx${new Guid()}`;
+            await appendElementTo('#container', 'div', id, {});
+
+            await createWidget('dxCheckBox', {
+              text,
+              value,
+              iconSize,
+              rtlEnabled,
+            }, `#${id}`);
+            await setClassAttribute(Selector(`#${id}`), state);
+          }
+        }
+      }
+    }
+  });
+});
+
+[45, undefined, 65].forEach((width) => {
+  test('Checkbox configuration, different Checkbox widths', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await testScreenshot(t, takeScreenshot, `CheckBox configurations width=${width === undefined ? 'default' : width}.png`, { element: '#container', shouldTestInCompact: true });
+
+    await testScreenshot(t, takeScreenshot, `CheckBox configurations width=${width === undefined ? 'default' : width}.png`, { element: '#container', theme: getDarkThemeName() });
+
+    await t
+      .expect(compareResults.isValid())
+      .ok(compareResults.errorMessages());
+  }).before(async () => {
+    await setStyleAttribute(Selector('#container'), 'padding: 5px; width: 300px; height: 400px; column-count: 2;');
```

Comment:
we have another test for column count, this is not a checkbox property, i suggest get rid of this styles. to crop label you can set checkboxes 'width' property. Anyway, if you need to test specific configurations ( styles ) for container, lets move it to separate test, eg 'test checkboxes appearance with differenc container configuration'.